tests/test_intl.py

```diff
@@ -150,6 +151,7 @@ def test_text_warning_node(app):
 
 
 @sphinx_intl
+@pytest.mark.skipif(platform.system() == 'Windows', reason='Skipped on windows')
```

Comment:
This would be better as `@pytest.mark.xfail(...)`, so that if someone does make it pass, appveyor will report it